src/stories/containers/users/user-profile/user-profile.tsx

```diff
@@ -0,0 +1,261 @@
+import { useRouter } from 'next/router';
+import React, { useCallback, useEffect, useLayoutEffect, useState } from 'react';
+import lightTheme from '../../../../../styles/theme/light';
+import { useAuthContext } from '../../../../core/context/AuthContext';
+import { useThemeContext } from '../../../../core/context/ThemeContext';
+import { ButtonType } from '../../../../core/enums/button-type.enum';
+import ControlledSwitches from '../../../components/button/switch-toogle/switch-component';
+import CloseButton from '../../../components/close-button/close-button';
+import { CustomButton } from '../../../components/custom-button/custom-button';
+import { CustomLink } from '../../../components/custom-link/custom-link';
+import AvatarPlaceholder from '../../../components/svg/avatar-placeholder';
+import { ENABLE_DISABLE_USER_REQUEST } from '../../auth/enable-disable-accounts/enable-disable.api';
+
+const arrayPassword = new Array<string>(8);
+const resultPassword = fill(arrayPassword, 'a');
+
+const UserProfile = () => {
+  const router = useRouter();
+  const { isLight } = useThemeContext();
+  const { user, clientRequest, clearCredentials } = useAuthContext();
+  const [checked, setChecked] = useState(false);
+  const isMobileOrTable = useMediaQuery(lightTheme.breakpoints.between('table_375', 'desktop_1194'));
+
+  const handleChange = useCallback(async () => {
+    const { query: gqlQuery, input } = ENABLE_DISABLE_USER_REQUEST(!checked, '1');
+    const data = await clientRequest?.request(gqlQuery, input);
+    if (data) {
+      console.log('data', data.userSetActiveFlag[0].active);
+      setChecked(data.userSetActiveFlag[0].active);
+    }
+  }, [checked, clientRequest]);
+
+  const handleDeleteAccount = useCallback(() => {
+    router.push('/auth/delete-account');
+  }, [router]);
+
+  const handleLogOut = () => {
+    clearCredentials && clearCredentials();
+  };
+
+  return (
+    <Wrapper isLight={isLight}>
+      <Container isLight={isLight}>
+        <ContainerInformation>
+          <CenterWrapper>
+            <AvatarPlaceholder />
+          </CenterWrapper>
+          <CenterWrapper>
+            <UserWrapper>
+              <LabelUser isLight={isLight}>{user?.username || ''}</LabelUser>
+              <UserRole>Site Admin</UserRole>
+            </UserWrapper>
+          </CenterWrapper>
+          <div
+            style={{
+              display: 'flex',
+              flexDirection: 'row',
+              alignItems: 'center',
+              marginBottom: '32px',
+            }}
+          >
+            <UserNameLabel isLight={isLight}>Username:</UserNameLabel>
+            <UserLabelValue isLight={isLight}>{user?.username}</UserLabelValue>
+          </div>
+          <ContainerPassword>
+            <UserNameLabel isLight={isLight}>Password:</UserNameLabel>
+            <div
+              style={{
+                display: 'flex',
+                flexDirection: 'row',
+                alignItems: 'center',
+              }}
```

Comment:
why no use styled-components in those divs instead of inline styles?

Reply:
You are right , I will use it